### ISSUE TITLE
[Dev-ready Design] Style guide for error types v3.0

### ISSUE THREAD

**Issue by reitermb** [raft-tech/TANF-app#1199]
**Description:**

Delivers a "style guide" for how errors in TDP should be _written_ (For Section 1 given the scope of v3.0)
- Error "templates"; anatomy of an error
- Important standardized terminology / other patterns of copy
- Principles for mapping how style guide elements combine in a holistic "error report"

**AC:**

- [ ] Documentation work for the following has occurred:
   - [ ] Relevant User stories.
   - [ ] Updating living UX documents, e.g. User Flows or Personas(if relevant).
- [ ] Dev/Design sync has occurred; resulting tickets created
- [ ] The design is usable and accessible, meaning it adheres to definition of done standards for design work.
> <details>
>      <summary>Expand Definition of Done Standards</summary>
> 
> - It uses [USWDS components and follows it’s UX guidance](https://designsystem.digital.gov/components/), or a deviation is clearly documented
> 
> - Language is intentional and [plain](https://plainlanguage.gov/guidelines/); placeholders are clearly documented
> - It follows [accessibility guidelines](https://accessibility.digital.gov/) and accessibility implementation notes are documented (e.g. clear information hierarchy, color is not the only way meaning is communicated, etc.)
> - If feedback identifies bigger questions or unknowns, create additional issues to investigate
> </details>

**Tasks:**

- [x] Create new Hack.md to draft style guide

**Supporting Documentation:**

[Hack.md](https://hackmd.io/@mreiter/S1HcLYQzo/edit)

**Comment by valcollignon** [comment]
@reitermb to create description and will bring to next UX backlog refinement to build AC with team
CC: @sreedevip

**Comment by valcollignon** [comment]
Additional description/AC/tasks to be decided pending roadmap meeting 12.14.21 and/or future research roadmap session(s)/UX refinements in the future.

**Comment by valcollignon** [comment]
Refinement of this ticket is blocked by roadmap refinement discussions. Per backlog refinement 12.21.21.
CC: @reitermb @sreedevip

**Comment by stevenino** [comment]
Not really a blocker to #1122 but it is a nice to have.

**Comment by stevenino** [comment]
@reitermb still require the updated supporting documentation.

**Comment by reitermb** [comment]
@ADPennington hack.md was giving me some issues so I'm spinning up a new one to transfer the content into—link in the description is updated!

**Comment by stevenino** [comment]
@reitermb requires minor edits.

**Comment by reitermb** [comment]
@ADPennington added the discussed clarity around the new dictionaries last week—the doc should be ready for another look!

**Comment by ADPennington** [comment]
> @ADPennington added the discussed clarity around the new dictionaries last week—the doc should be ready for another look!

thank you @reitermb this is approved 👍🏾
